scripts/index.js

```diff
@@ -0,0 +1,44 @@
+var blog =
+$('article:first').clone()[0];
+
+article.prototype.toHTML = function (argument) {
+  $artSkel = $('article').filter(':first').clone();
+};
+
+function postArticle (rawArticle) {
+  var art = new ARticle(rawArticle);
+  art.toHTML();
+};
+
+var Article = function (props) {
+  this.author = props.authoer;
+  this.whatever = props.whatever;
+  articles.push(this);
+};
+
+Article.prototype.toHTML = function() {
+  var $clonedARticle = $('article#template').clone();
+  $clonedArticle.removeAttr('id');
+  $clonedArticle.find('h1.author').html('<a href "' + articles[i].authorURL + '" ' + articles[i].author);
```

Comment:
This is a bit awkward to be inserting html from here. What about using  `$clonedArticle.find('...').attr('href', this.authorUrl);`